### PR TITLE
Amazon Web Services (AWS) vs Google Cloud

### LINKED ISSUE #726: Amazon Web Services (AWS) vs Google Cloud
### Brief Summary: 
This article provides a broad comparison of two giant cloud computing giants: Amazon Web Services (AWS) and Google Cloud.

### Key Takeaways: 
1. Definition of AWS and Google Cloud
2. Features of these services
3. Key differences between AWS and Google Cloud
4. Advantages and disadvantages of these cloud computing services

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
This fixes #726 .

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/aws-vs-google-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+* They have a low cost of ownership compared to dedicated or private servers.
+* The AWS platform has hybrid capabilities that give it a cutting edge over its competitors.
+* Billing and management in the AWS platform are centralized.
+* In AWS, applications can be deployed in multiple regions around the globe.
+* AWS has an introduction feature called Identity and Access Management (IAM). This feature enhances the creation of groups or users. 
+* It enhances virtual privacy through the provision of a Virtual Private Cloud (VPC) that does not interfere with other networks. 
+
+### Features of Google Cloud
+* It allows the constant inclusion of more languages.
+* In Google Cloud platforms, user experience can be improved if a better user interface is used.
```
Comment (adrianmurage) [inline articles/aws-vs-google-cloud/index.md:35]: I don't think this can be considered a feature

File: articles/aws-vs-google-cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* It can enhance broad network access. 
+* Scalable services can be provided by Google Cloud platforms.
+* It allows cloud resource pooling- This is a service that enables the provider to serve many customers with provisional services. Adjustments can be made to these services to meet the demand of customers.
+  
+### Differences Between AWS and Google Cloud
+AWS and Google Cloud can be distinguished by key parameters such as users, compute services, storage, Big Data analysis, networking, features, downtimes and pricing. 
+
+#### Users
+AWS has been able to attract high-profile clients because it is established, and has extensive experience in cloud computing. AWS platform is used by clients such as Unilever, Samsung, Netflix, Xiami, BMW, and Airbnb.
```
Comment (adrianmurage) [inline articles/aws-vs-google-cloud/index.md:44]: I think you meant Xiaomi?

File: articles/aws-vs-google-cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* It offers a wide variety of services, with the capability of handling many resources and users.
+  
+#### Advantages of Google Cloud
+* It provides quality computing services in Big Data, machine learning, and analytics.
+* It offers considerable load balancing and scaling.
+* It has good pricing plans. 
+* They enhance the live migration of VMs (Virtual Machines).
+
+#### Disadvantages of AWS
+* It has a long deployment process. This process can take up to 20 minutes for one website.
```
Comment (adrianmurage) [inline articles/aws-vs-google-cloud/index.md:88]: In my experience deployment processes are not tied to the cloud platform one uses.
Kindly add statistics to this claim or consider removing it altogether

File: articles/aws-vs-google-cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+* It has good pricing plans. 
+* They enhance the live migration of VMs (Virtual Machines).
+
+#### Disadvantages of AWS
+* It has a long deployment process. This process can take up to 20 minutes for one website. 
+* It may not be ideal for companies that do not have a high technological capability.
+* The cost structure is difficult to understand.
+  
+#### Disadvantages of Google Cloud
+* The Google Cloud lacks certain features that exist in the AWS platform.
```
Comment (adrianmurage) [inline articles/aws-vs-google-cloud/index.md:93]: Mentioning these features would be ideal

File: articles/aws-vs-google-cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+* Its user interface does not offer adequate customer support.
```
Comment (adrianmurage) [inline articles/aws-vs-google-cloud/index.md:94]: Kindly elaborate?

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@mbaabuones 
An overall good article however some of the claims need a basis or some statistical facts to back them up. Consider linking to published statistics or swapping them out in favor of things that have the statistical backings.

Once you've made updates to the article ping me, I'll have a final look and mark the article ready for publishing

File: articles/aws-vs-google-cloud/index.md (hunk 1 above)
Comment (mbaabuones) [reply articles/aws-vs-google-cloud/index.md]: I have deleted and replaced it with another good feature.

File: articles/aws-vs-google-cloud/index.md (hunk 2 above)
Comment (mbaabuones) [reply articles/aws-vs-google-cloud/index.md]: Yes. Thanks for the correction. I have changed it.

File: articles/aws-vs-google-cloud/index.md (hunk 3 above)
Comment (mbaabuones) [reply articles/aws-vs-google-cloud/index.md]: I have deleted this.

File: articles/aws-vs-google-cloud/index.md (hunk 4 above)
Comment (mbaabuones) [reply articles/aws-vs-google-cloud/index.md]: Done.

File: articles/aws-vs-google-cloud/index.md (hunk 5 above)
Comment (mbaabuones) [reply articles/aws-vs-google-cloud/index.md]: I have also deleted this.

**Comment by mbaabuones** [comment]
@adrianmurage Thank you for your suggestions. I have finished working on the changes.

**Review by adrianmurage — APPROVED** [review]

**Comment by adrianmurage** [comment]
@hectorkambow 
This article is ready for final review 

@mbaabuones Great job with this!
Looking forward to seeing greater articles

**Comment by hectorkambow** [comment]
@mbaabuones great article - learning quite a bit about these cloud providers - moving forward I think it would be helpful to add more links in your articles. 👍 
Thank you

**Pull request merged**